resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/K8sVolumeTestUtils.scala

```diff
@@ -20,7 +20,7 @@ import scala.collection.JavaConverters._
 
 import io.fabric8.kubernetes.api.model.{Container, Pod}
 
-private[spark] object SecretVolumeUtils {
+private[spark] object K8sVolumeTestUtils {
```

Comment:
This refactoring is completely orthogonal to this PR. Let's not renaming in this PR.